.changeset/dull-pens-grow.md

```diff
@@ -0,0 +1,11 @@
+---
+"@kaizen/draft-tooltip": patch
+---
+
+Updates tooltip to pass accessible descriptions only into native or custom somantic components and only when in the DOM
```

Comment:
```suggestion
Updates tooltip to pass accessible descriptions only into native or custom semantic components and only when in the DOM
```

Reply:
really need to reinstall that spell check plugin 😆